lib/src/main/java/com/futurewei/alcor/common/entity/CustomerResource.java

```diff
@@ -17,12 +17,14 @@
 package com.futurewei.alcor.common.entity;
 
 import com.fasterxml.jackson.annotation.JsonProperty;
+import org.apache.ignite.cache.query.annotations.QuerySqlField;
 
 import java.util.Objects;
 
 public class CustomerResource extends Resource {
 
     @JsonProperty("project_id")
+    // @QuerySqlField(index = true)
```

Comment:
@pkommoju If we don't need add `@QuerySqlField` here, please remove it and the import.

Reply:
@DavidLiu506 Please confirm, if we need the index on `project_id`?

Reply:
I will remove it after Dahai confirms that it is no longer needed.

Reply:
Removed.